src/meshapi/management/commands/replay_join_records.py

```diff
@@ -42,18 +42,27 @@ def add_arguments(self, parser: ArgumentParser) -> None:
         )
 
     def handle(self, *args: Any, **options: Any) -> None:
-        logging.info("Fetching Join Records...")
+        # Default to getting join records from 1 week ago unless otherwise specified
+        since = options["since"] or self.past_week()
```

Comment:
Does `options["since"]` need to be converted to `datetime`?

Reply:
There is a [lambda](https://github.com/nycmeshnet/meshdb/pull/836/files#diff-aed32b816222e67d35fdcf17ed956a53bb18f387e96dd5b6cb22c6d3daa91d53R34) on the command line argument that converts it to datetime.